lib/fluent/log.rb

```diff
@@ -338,13 +345,13 @@ def event(level, args)
         end
       }
 
-      unless @threads_exclude_events.include?(Thread.current)
+      if @log_event_enabled && !@threads_exclude_events.include?(Thread.current)
         record = map.dup
         record.keys.each {|key|
           record[key] = record[key].inspect unless record[key].respond_to?(:to_msgpack)
         }
         record['message'] = message.dup
-        @engine.push_log_event("#{@tag}.#{level}", time.to_i, record)
+        @engine.push_log_event("#{LOG_EVENT_TAG_PREFIX}.#{level}", time.to_i, record)
```

Comment:
Right... I added a commit to emit events with `Fluent::EventTime.from_time(time)`.